docs/teams/mentoring.md

```diff
@@ -1,68 +1,174 @@
----
-original:
-  authors: Fiona Gillan
-  url: https://github.com/fgillan/sr-team-coord/wiki/Mentoring
----
 # Mentoring
 
-* Contact the Volunteer Coordinator asking for potential Mentors.
-* A Google Form can be used to obtain the following information:
-    * Confirm over 18
-    * Name
-    * Email address
-    * Location
-    * How far willing to travel to mentor
-    * Roughly how often could commit to visiting a team
-    * What skills do you possess making you suitable mentor
-    * Previous SR experience
-
-## Example of email sent to potential Mentors asking for volunteers in 2017
```

Comment:
Thoughts on capturing this email content somewhere?

Reply:
Yes: https://github.com/srobo/volunteer-emails/pull/43 does this.